backend/app/views/spree/admin/orders/index.html.erb

```diff
@@ -65,14 +65,21 @@
             </div>
 
             <div class="col-12 col-xl-6">
-              <div class="field">
-                <%= label_tag :q_bill_address_firstname_start, t('spree.first_name_begins_with') %>
-                <%= f.text_field :bill_address_firstname_start, size: 25 %>
-              </div>
-              <div class="field">
-                <%= label_tag :q_bill_address_lastname_start, t('spree.last_name_begins_with') %>
-                <%= f.text_field :bill_address_lastname_start, size: 25%>
-              </div>
+              <% if Spree::Config.use_combined_first_and_last_name_in_address %>
+                <div class="field">
+                  <%= label_tag :q_bill_address_name_start, t('spree.name_contains') %>
```

Comment:
Should this be `q_bill_address_name_cont`, right?

Reply:
Not exactly, this part is bit tricky. 
`name` is a virtual attribute and we can't have a ransacker working on it and, to have the least impact on how the system works right now, the search on name is just as an alias of searching by `_firstname_start` or `_lastname_start`.
So it's neither a `_start` nor a `_cont`, but I opted to keep `_start` to keep it close to the ransackers it's aliasing. 
We could use `_firstname_or_startname_start` but it's quite verbose, I'm not sure which would be the best solution..

Reply:
Got it, my personal preference is `_firstname_or_lastname_start`